client/main.lua

```diff
@@ -175,7 +170,33 @@ local function findKeys(vehicle, plate)
     SetTimeout(10000, function()
         attemptPoliceAlert('steal')
     end)
+end
 
+local function showHotwiringLabel()
+    if isHotwiring then return end
+    isHotwiring = true
+    CreateThread(function()
+        -- Hotwiring while in vehicle, also keeps engine off for vehicles you don't own keys to
+        while LocalPlayer.state.isLoggedIn and cache.vehicle do
```

Comment:
```suggestion
        while cache.vehicle do
```
I don't think a player could be not logged in while in a vehicle in normal cases.